src/app/shared/top-panel/search-box/input/suggestions/model/suggestions.util.spec.ts

```diff
@@ -0,0 +1,175 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {CollectionModel} from '../../../../../../core/store/collections/collection.model';
+import {convertSuggestionsToQueryItemsSorted} from './suggestions.util';
+import {ViewModel} from '../../../../../../core/store/views/view.model';
+import {Perspective} from '../../../../../../view/perspectives/perspective';
+import {LinkTypeModel} from '../../../../../../core/store/link-types/link-type.model';
+import {CollectionQueryItem} from '../../../query-item/model/collection.query-item';
+import {AttributeQueryItem} from '../../../query-item/model/attribute.query-item';
+import {ViewQueryItem} from '../../../query-item/model/view.query-item';
+import {LinkQueryItem} from '../../../query-item/model/link.query-item';
+import {FulltextQueryItem} from '../../../query-item/model/fulltext.query-item';
+
+const collections: CollectionModel[] = [
+  {
+    id: 'c1',
+    name: 'lumeer',
+    attributes: [],
+  },
+  {
+    id: 'c2',
+    name: 'sports',
+    attributes: [],
+  },
+  {
+    id: 'c3',
+    name: 'whey',
+    attributes: [],
+  },
+];
+const collectionQueryItems = collections.map(c => new CollectionQueryItem(c));
+
+const attributes: CollectionModel[] = [
+  {
+    ...collections[0],
+    attributes: [{id: 'a1', name: 'a1', constraints: []}],
+  },
+  {
+    ...collections[1],
+    attributes: [{id: 'a2', name: 'a2', constraints: []}],
+  },
+  {
+    id: 'lala',
+    name: 'lala',
+    attributes: [{id: 'a2', name: 'a2', constraints: []}],
+  },
+];
+const attributeQueryItems = attributes.map(a => new AttributeQueryItem(a, a.attributes[0], '', ''));
+
+const views: ViewModel[] = [
+  {
+    id: 'v1',
+    name: 'v1',
+    perspective: Perspective.Search,
+    config: {},
+    query: {},
+  },
+  {
+    id: 'v2',
+    name: 'v2',
+    perspective: Perspective.Table,
+    config: {},
+    query: {},
+  },
+];
+const viewsQueryItems = views.map(v => new ViewQueryItem(v));
+
+const linkTypes: LinkTypeModel[] = [
+  {
+    id: 'l1',
+    name: 'l1',
+    collectionIds: [collections[0].id, collections[1].id],
+    collections: [collections[0], collections[1]],
+  },
+  {
+    id: 'l2',
+    name: 'l2',
+    collectionIds: [collections[2].id, collections[1].id],
+    collections: [collections[2], collections[1]],
+  },
+];
+const linkTypeQueryItems = linkTypes.map(l => new LinkQueryItem(l));
+
+const fulltextQueryItems = [new FulltextQueryItem('la'), new FulltextQueryItem('he')];
+
+fdescribe('Suggestions util', () => {
+  fit('should filter undefined suggestions', () => {
```

Comment:
You probably forgot to change this back to `describe` and `it`. Other tests are ignored now.